test/groovy/vars/OdsComponentStageRolloutOpenShiftDeploymentSpec.groovy

```diff
@@ -187,18 +191,17 @@ class OdsComponentStageRolloutOpenShiftDeploymentSpec extends PipelineSpockTestB
     }
     def deploymentInfo = script.call(context)
 
+    // We really only care about the call to `helmUpgrade'
     then:
     printCallStack()
     assertJobStatusSuccess()
-    deploymentInfo['DeploymentConfig/bar'][0].deploymentId == "bar-124"
```

Comment:
I am surprised this was removed ... I would expect that we get a deployment back ... ?